doc/_static/mpl.css

```
@@ -185,3 +187,11 @@ div.wide-table table th.stub {
 .sdd.sd-dropdown .sd-card-header +.sd-card-body{
     --pst-sd-dropdown-color: none;
 }
+
+/*removes page title from toctree listing*/
```

Can you add where this is needed? It feels like a workaround for missing ReST/Sphinx functionality. I'm slightly concerned that this may affect other places where it's not intended.

---

Ah, I see. `section-toc` is a custom class that can be added to toctrees.

```suggestion
/* section-toc is a custom class that can be used to remove the page title from a toctree listing
 * Example usage:
 *
 *     .. rst-class:: section-toc
 *
 *        .. toctree::
 */
```